app/src/main/java/com/example/henzoshimada/feeltrip/FeelTripApplication.java

```diff
@@ -41,6 +39,68 @@ public static String getEmotionalState(int index){
         return allEmotions.get(index);
     }
 
+
+    // THEMES HERE
+    private static int themeID = R.style.DefaultTheme;
+
+    public static int getThemeID() {
+        return themeID;
+    }
+
+    public static void setThemeID(int themeID) {
+        FeelTripApplication.themeID = themeID;
+    }
+
+    public static int getCOLORPRIMARY() {
+        return COLORPRIMARY;
+    }
+
+    public static void setCOLORPRIMARY(int COLORPRIMARY) {
+        FeelTripApplication.COLORPRIMARY = COLORPRIMARY;
+    }
+
+    public static int getTEXTCOLORPRIMARY() {
+        return TEXTCOLORPRIMARY;
+    }
+
+    public static void setTEXTCOLORPRIMARY(int TEXTCOLORPRIMARY) {
+        FeelTripApplication.TEXTCOLORPRIMARY = TEXTCOLORPRIMARY;
+    }
+
+    public static int getTEXTCOLORSECONDARY() {
+        return TEXTCOLORSECONDARY;
+    }
+
+    public static void setTEXTCOLORSECONDARY(int TEXTCOLORSECONDARY) {
+        FeelTripApplication.TEXTCOLORSECONDARY = TEXTCOLORSECONDARY;
+    }
+
+    public static int getTEXTCOLORTERTIARY() {
+        return TEXTCOLORTERTIARY;
+    }
+
+    public static void setTEXTCOLORTERTIARY(int TEXTCOLORTERTIARY) {
+        FeelTripApplication.TEXTCOLORTERTIARY = TEXTCOLORTERTIARY;
+    }
+
+    private static int COLORPRIMARY = 0xFF3F51B5;
+
+    private static int TEXTCOLORPRIMARY = 0xFF3F51B5;
+
+    private static int TEXTCOLORSECONDARY = 0xFF3F51B5;
+
+    private static int TEXTCOLORTERTIARY = 0xFF3F51B5;
```

Comment:
These initializations here really don't serve to do anything. It just has to be initialized to something, and I used DARK_BLUE to test it out.